Samples/OpenQasmReader.Tests/FirstLetterToUpperCaseTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
```

Comment:
please add MIT license headers (for all .cs source).

Reply:
executed